website/docs/getting-started/setup/installation-guides/docker/README.md

```diff
@@ -5,172 +5,157 @@ sidebar_position: 1
+<div class="downloadAssets">
+    <a target="_blank" href='/img/docker-compose_(3).yml' download="docker-compose_(3).yml">
+     <img src="/img/FileDownload.png" alt="Click to Download"/>
+    </a>
+</div>
 
-**or** run the following curl if you're on a remote machine:
+**OR,** run the following cURL:
 
 ```bash
 curl -L https://bit.ly/3AQzII6 -o $PWD/docker-compose.yml
 ```
 
-:::note
-For the **Business Edition** - change the **image** name from `appsmith-ce` to `appsmith-ee` for the `image:` key in the `docker-compose.yml` file.
-:::
+:::info
+**For Business Edition users:**
 
-This configuration runs an Appsmith instance and a Watchtower instance to keep Appsmith automatically up-to-date.
+1. Change the **image** name from `appsmith-ce` to `appsmith-ee` for the `image:` key in the `docker-compose.yml` file.
 
-:::note
-For the **Business Edition** - add your **license key** (`APPSMITH_LICENSE_KEY`) to the `environment:` key in the `docker-compose.yml` file.
+2. Add your **license key** (`APPSMITH_LICENSE_KEY`) to the `environment:` key in the `docker-compose.yml` file.
+
+To upgrade your existing Community Edition installation to the Business Edition, [follow these instructions](../../upgrade-to-business-edition/).
 :::
 
-Bring the docker container up by running the following command. (You may need to run as `sudo` if docker and docker-compose are not accessible by your user)
+Start up the Docker container by running the following command - You may need to run with `sudo` if docker and docker-compose aren't accessible by your user.
 
 ```bash
 docker-compose up -d
 ```
 
-If it is not available locally, the command above will download the Docker images and start the services. You can follow the logs with the following command:
+If the image doesn't exist locally, this command downloads the Docker images and starts its services. You can follow the logs with the following command:
 
 ```bash
 docker logs -f appsmith
 ```
 
-You should see a message `Appsmith is Running!` once the container is ready. The message is also logged and available in server logs(<mark>`stacks/logs/backend/backend.log`</mark>).
+Once the container is ready, you should see `Appsmith is Running!` in the Docker logs. This message is also logged and available in server logs in (<mark>`stacks/logs/backend/backend.log`</mark>).
 
 ![Appsmith is running message](/img/InstallationGuides__Docker__AppsmithRunningMessage.png)
 
-:::tip
-If you are **unable** to **locate** the message but can **access** Appsmith via **localhost** or a **custom domain** you have **created**, then Appsmith is up and running.
-:::
-
-If this is your first time using docker, you should expect a welcome page similar to the one below.
+Congratulations, your Appsmith server should be up and running now. You can access it at [http://localhost](http://localhost). If this is your first time setting up using Docker, you should expect a welcome page similar to the one below.
```

Comment:
The congratulations here seems unnecessary. IMO 'The Appsmith instance should be up and running now' should suffice.